packages/webapp/src/components/Notifications/NotificationTimeline/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import styles from './styles.module.scss';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { getNotificationCardDate } from '../../../util/moment';
+import useTranslationUtil from '../../../util/useTranslationUtil';
+import { useTranslation } from 'react-i18next';
+import history from '../../../history';
+import MoreRecentNotificationWarning from '../MoreRecentNotificationWarning';
+
+function NotificationTimeline({ activeNotificationId, relatedNotifications, style }) {
+  const { t } = useTranslation();
+  const { getNotificationTitle } = useTranslationUtil();
+  const [sortedRelatedNotifications, setSortedRelatedNotifications] = useState([]);
+  const [mostRecentNotification, setMostRecentNotification] = useState(null);
+
+  useEffect(() => {
+    setMostRecentNotification(null);
+    setSortedRelatedNotifications(
+      relatedNotifications.sort((a, b) => {
+        return new Date(b.created_at) - new Date(a.created_at);
+      }),
+    );
+  }, [activeNotificationId]);
+
+  useEffect(() => {
+    if (
+      sortedRelatedNotifications &&
+      sortedRelatedNotifications[0]?.notification_id !== activeNotificationId
+    ) {
+      setMostRecentNotification(sortedRelatedNotifications[0]);
+    }
+  }, [sortedRelatedNotifications]);
+
+  return (
+    <div className={styles.container} style={style}>
+      <header className={styles.header}>
+        <h2 className={styles.heading}>{t('NOTIFICATION.TIMELINE.HEADING')}</h2>
+        <hr className={styles.headerLine} />
+      </header>
+      {mostRecentNotification && (
+        <MoreRecentNotificationWarning
+          notificationId={mostRecentNotification?.notification_id}
+          entityType={mostRecentNotification?.ref?.entity?.type}
```

Comment:
I think the first optional chaining on this and the previous line are redundant with the check on lint 41?